components/windows-service/plan.ps1

```diff
@@ -1,6 +1,6 @@
 $pkg_name="windows-service"
 $pkg_origin="core"
-$pkg_version="0.5.0"
+$pkg_version="0.6.0"

```

Comment:
Out of curiosity, do we have any kind of meaningful versioning scheme set up for this particular package yet?

Reply:
no we don't. Now that we are moving to releasing one with every release, perhaps we should just user the `VERSION` file and pull it in to the overall habitat versioning.

Reply:
thinking more on this, I vote to keep this as is for now. If we moved to the larger hab versioning that carries with it the assumption that a 0.90.0 windows-service would be tied to a 0.90.0 supervisor which is not true.